.github/workflows/lint.yml

```diff
@@ -2,23 +2,24 @@ name: Lint
 
 on:
   push:
+    # On other branches the `pull_request` trigger will be used
+    branches: [current, next]
+
   pull_request:
+    # Only trigger on certain events (not when comments are added)
+    types: [opened, reopened, synchronize]
```

Comment:
these are default https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

```suggestion
```

Reply:
Oops, mea culpa. Thanks!